scenarios/ScenarioFoundFood.py

```diff
@@ -0,0 +1,18 @@
+import random
+import scenarioManager
+    
+class ScenarioFoundFood(scenarioManager.Scenario):
+    def __init__(self, gui, player, name, description, choice):
+        super().__init__(gui, player, name, description, choice)
+
+    def run(self):
+        print(f"Running scenario {self.name}...")
+        print(self.description)
+        
+        self.mod.distance = random.randint(3, 5)
+        print("You found edible dead pythons in your food supplies. You gained  {self.mod.distance} food units.")
```

Comment:
You should have a line that writes the result in the mod object like this
self.mod.result = f"You found edible dead pythons in your food supplies. You gained  {self.mod.distance} food units."
Also make sure to include the f before the string